7.9 release notes (#3903)
astronomer/docs: astro/release-notes.md

<!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->

## July 9, 2024

### Additional improvements

- Fixed a bug that prevented adding a date input for a DAG filter in the Astro UI.

## July 2, 2024

### Export metrics about your Astro Deployments to observability tools

<TeamBadge/>

:::publicpreview
:::

> You can now export comprehensive, operational metrics about the performance of your Astro Deployments to third-party observability tools, such as New Relic, using the new universal Metrics Exporter. This new feature allows you to configure an endpoint to send Astro metrics using the Prometheus data model, which means you can integrate your Astro observability metrics directly into your existing monitoring tools. See [Export metrics from Astro](https://www.astronomer.io/docs/astro/export-metrics) for setup instructions.

Reviewer: ```suggestion
You can now export comprehensive, operational metrics about the performance of your Astro Deployments to third-party observability tools, such as New Relic, using the new Universal Metrics Exporter. This new feature allows you to configure a Prometheus endpoint to export metrics using the Prometheus data model, which means you can integrate your Astro observability metrics directly into your existing monitoring tools. See [Export metrics from Astro](https://www.astronomer.io/docs/astro/export-metrics) for setup instructions.
```
